.eslintrc

```diff
@@ -10,15 +10,20 @@
     "react/prefer-es6-class": 0,
     "react/prefer-stateless-function": 0,
     "react/no-multi-comp": 0,
-    "array-callback-return": "off", 
+    "array-callback-return": "off",
     "arrow-body-style": ["warn", "as-needed"],
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "always"],
```

Comment:
Now that we've got into the habit of removing dangling commas, would it be annoying to ask people to start adding them back in again?